aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -495,6 +500,12 @@ fn native_string_concat(
 
     let prefix = string_to_bytes(safely_pop_arg!(args, Struct))?;
 
+    if prefix.len() + suffix.len() > CONCAT_PREFIX_AND_SUFFIX_MAX_LENGTH {
```

Comment:
Shouldn't we be checking that the length of Prefix + Snapshot + Suffix <= 256?

Reply:
I don't think we can check it because then we read the snapshot value? Unless it is a speculative read

Reply:
In that case we can check Prefix + 8 + Suffix <= 256, assuming snapshot length is at most 8 bytes?

Reply:
u128 is up to 20 characters.

but why to sum up, it's just a random constant? since I want it to be predictable - having it explicit on prefix and suffix seems cleaner.